nginx/timings.py

```diff
@@ -96,3 +125,8 @@ def _sanitize_url(url):
     # Remove URL params
     url = re.sub(r'\?[^ ]*', '', url)
     return url
+
+
+def _extract_domain(url):
+    match = re.search(r'/a/(?P<domain>[0-9a-z-]+)', url)
```

Comment:
doesn't matter. the domain capture will stop at `/` since it isn't in the specified characters. i only prefixed with `/a/` so i was sure i was getting the domain and not another string that looked similar